admin-dev/themes/new-theme/js/fos_js_routes.json

```diff
@@ -1 +1 @@
,"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_send_process_order_email":{"tokens":[["text","\/sell\/orders\/process-order-email"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_add_product":{"tokens":[["text","\/products"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_delete_product":{"tokens":[["text","\/delete"],["variable","\/","\\d+","orderDetailId"],["text","\/products"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+","orderDetailId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_get_discounts":{"tokens":[["text","\/discounts"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_get_prices":{"tokens":[["text","\/prices"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_get_payments":{"tokens":[["text","\/payments"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_get_products":{"tokens":[["text","\/products"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_get_invoices":{"tokens":[["text","\/invoices"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_get_documents":{"tokens":[["text","\/documents"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_cancellation":{"tokens":[["text","\/cancellation"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_configure_product_pagination":{"tokens":[["text","\/sell\/orders\/configure-product-pagination"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_product_prices":{"tokens":[["text","\/products\/prices"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_orders_products_search":{"tokens":[["text","\/sell\/orders\/products\/search"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["GET"],"schemes":[]}},"prefix":"","host":"localhost","port":"","scheme":"http","locale":[]}
```

Comment:
It was rebuilt 😄  but I did not highlight this in a commit. I did it "on the go" while resolving conflicts.

Reply:
OK I wasn't sure ^^ It's because I do it differently when I merge, first resolve conflicts by keeping develop file Then add a commit which rebuilds it
Both techniques are valid!